css/properties/animation.json

```diff
@@ -0,0 +1,149 @@
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "edge_mobile": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "16"
+              },
+              {
+                "prefix": "-moz-",
+                "version_added": "5"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "49"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "44",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.prefixes.webkit",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "16"
+              },
+              {
+                "prefix": "-moz-",
+                "version_added": "5"
+              },
+              {
+                "version_added": "49",
+                "prefix": "-webkit-"
+              },
+              {
+                "version_added": "44",
+                "prefix": "-webkit-",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.prefixes.webkit",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": [
+              {
+                "version_added": "12"
+              },
+              {
+                "prefix": "-o-",
+                "version_added": true,
+                "version_removed": "12.50",
+                "notes": "See the <a href='http://my.opera.com/ODIN/blog/2012/08/03/a-hot-opera-12-50-summer-time-snapshot'>Opera 12.50 release notes</a>."
+              }
+            ],
+            "safari": {
+              "prefix": "-webkit-",
+              "version_added": "4"
+            },
+            "safari_ios": {
+              "prefix": "-webkit-",
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "frames_timing_function": {
+          "__compat": {
+            "description": "<code>frames</code> timing function",
+            "support": {
```

Comment:
Add other mobile versions, with false. (ie and ie_mobile must have false there too)